update.go

```diff
@@ -254,6 +260,17 @@ other options are ignored.
 		config.Cgroups.Resources.MemorySwap = *r.Memory.Swap
 		config.Cgroups.Resources.PidsLimit = r.Pids.Limit
 
+		if val := context.String("l3-cache-schema"); val != "" {
+			// Avoid silent failure in container.Set()
+			if !intelrdt.IsEnabled() || config.IntelRdt == nil {
+				return fmt.Errorf("l3 cache schema is not enabled or not specified in config, update value '%s' is discarded", val)
+			}
+			if config.IntelRdt == nil {
```

Comment:
@crosbymichael 
This if condition `if config.IntelRdt == nil` is always false, because error returns above when `config.IntelRdt == nil`.
I will remove this if condition lines.

Then the code will look like:
```
        Action: func(context *cli.Context) error {
                ...

                if val := context.String("l3-cache-schema"); val != "" {
                        // Avoid silent failure in container.Set()
                        if !intelrdt.IsEnabled() || config.IntelRdt == nil {
                                return fmt.Errorf("l3 cache schema is not enabled or not specified in config, update value '%s' is discarded", val)
                        }
                        config.IntelRdt.L3CacheSchema = val
                }

                return container.Set(config)
        },
```